### ISSUE TITLE
[JOSS] Nitpicks on the paper

### ISSUE THREAD

**Issue by adswa** [mne-tools/mne-icalabel#107]
As part of the [JOSS review](https://github.com/openjournals/joss-reviews/issues/4484) I stumbled over some paragraphs or wordings in the paper. All of them are nitpicks that you can feel free to ignore, and the editors comments would take complete precedence - so please feel free to close this issue any time, and @emdupre can come in and invalidate any of my comments as she sees fit as well without explanation. 

>  However, this package was previously only available in MATLAB, limiting its usage among Python neuroscience researchers.

The term "Python neuroscience researchers" is certainly understandable, but somehow still feels like a slightly weird composite term to me (it boils down to "Python researchers"). Given that the reduced accessibility of closed-source MatLab software is known in the open source world, I think the sentence would convey the same meaning and intention without the second half of the sentence - or, alternatively, you could make a slight change like 'neuroscientists preferring Python-based tools' to avoid the "Python researchers".

> MNE-Python is a general-purpose electrophysiology analysis package in Python that has a large core group of developers. Integration into MNE makes it likely that MNE-ICALabel will be maintained and continue to be improved. MNE-Python also has stability due to the funding it receives directly for development from institutions such as National Institutes of Health, the Chan Zuckerberg open-source initiative, the European Research Council and Agence Nationale de la Recherche in France.

I think this is a strong paragraph highlighting a major advantage of this package and further warranting the porting of ICLabel into this cool open source tool. I believe that from a structural perspective it would better fit one paragraph up, right into the end of the paragraph starting with "ICLabel was a proposed statistical model that uses neural networks and a crowdsourced training dataset to automatically label ICA components [@iclabel2019]. However, this package [...]" that highlights the other advantages this package has over the EEGLab alternative/origin.

> In future versions, we plan on building on top of the ICLabel model to improve its robustness when operating on different types of recording hardware, sensor type, and sensor count. Moreover, we plan on building new types of models that improve their overall accuracy and performance with respect to auto-labeling neural and non-neural signals. The availability of a simple API for ICLabel will facilitate a strong benchmark to build future models and bring in new developers and contributors.

I kept wondering whether this paragraph might better fit into a new section "Development goals" or "Future directions" rather than "Statement of Need", but I am unsure. In general, it feels like this paragraph interrupts the line of thought in the previous and preceeding paragraph (i.e., my comment above), and it feels like the last sentence ("The developer team is excited to improve the state of the art in data handling for ICA preprocessing and looking forward to welcoming new contributors and users from the broader MNE, neuroscience and electrophysiology community.") would fit more nicely at the end of this paragraph.

Again, feel free to disagree with any of those comments and close this issue at any time.

**Comment by adam2392** [comment]
Thanks @adswa for all these comments. I agree w/ all of them and have added them into the paper.
